guiEditor/src/tools.ts

```diff
@@ -41,16 +41,17 @@ export class Tools {
         if (parent && parent.reservedDataStore && parent.reservedDataStore.detachedChildren) {
             finalArray.push(...parent.reservedDataStore.detachedChildren);
         }
+        return finalArray.reverse();
 
         return finalArray.sort((a: any, b: any) => {

```

Comment:
This is useless if you return 2 lines before

Reply:
Aha forgot to remove ;) Good find.